internal/prettier/prettier.go

```diff
@@ -0,0 +1,45 @@
+package prettier
+
+import (
+	"errors"
+	"fmt"
+	"os/exec"
+	"strings"
+	"sync"
+)
+
+var resolveFormatter = sync.OnceFunc(func() {
+	candidates := [][]string{
+		{"prettierd"},
+		{"prettier"},
+		{"npx", "prettier"},
```

Comment:
I wonder if this can be expanded some, perhaps via configuration. Let's say the dev has a `frontend` folder in their repository:

```tree
my-app/
├─ templates/
│  ├─ root.templ
├─ go.mod
├─ main.go
├─ frontend/
│  ├─ node_modules/
│  ├─ package.json
│  ├─ prettier.config.js
```

If we give them a way to pass in the dir, prettier can be executed in that context, with all plugins that they might have.

Reply:
Sounds reasonable, something like`templ fmt --prettier-cmd="npx prettier" --prettier-config=./frontend`?

Reply:
`--use-tabs` is an interesting tidbit, because it really should be enabled (like you have it here), but maybe it's just a bit of documentation.